container/container_unix.go

```diff
@@ -22,8 +22,11 @@ import (
 	"github.com/opencontainers/runc/libcontainer/label"
 )
 
-// DefaultSHMSize is the default size (64MB) of the SHM which will be mounted in the container
-const DefaultSHMSize int64 = 67108864
+const (
+	// DefaultSHMSize is the default size (64MB) of the SHM which will be mounted in the container
+	DefaultSHMSize           int64 = 67108864
```

Comment:
What is this used for? Does this mean that for each `tmpfs` we're effectively paying 64Mb?